Webshop.Catalog.Solution/MessagingService/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using RabbitMQ.Client;
+using System;
+using System.Text;
+
+class Program
+{
+    static void Main()
+    {
+        var factory = new ConnectionFactory() { HostName = "localhost" };
+
+        using (var connection = factory.CreateConnection())
+        using (var channel = connection.CreateModel())
+        {
+            channel.QueueDeclare(queue: "your_queue_name", durable: false, exclusive: false, autoDelete: false, arguments: null);
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏It might be best if we give the `your_queue_name` a better name to better reflect the purpose of the queue.

Reply:
its a placeholder only for now and is to be updated in near future once I get to work on the RabbitMQ a little more again